aea/helpers/async_utils.py

```diff
@@ -361,73 +360,6 @@ def stop(self) -> None:
         _default_logger.debug("Stopped.")
 
 
-class AwaitableProc:
```

Comment:
Why removed?

Reply:
not used at all, some time ago i replaced this class with std library solution